packages/firebase_vertexai/firebase_vertexai/test/live_test.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'dart:typed_data';
+
+import 'package:firebase_vertexai/src/content.dart';
+import 'package:firebase_vertexai/src/error.dart';
+import 'package:firebase_vertexai/src/live_api.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:web_socket_channel/web_socket_channel.dart';
+
+// Mock classes for dependencies
+class MockWebSocketChannel extends Mock implements WebSocketChannel {}
```

Comment:
Is this unused? If we need a test version of a websocket we should be able to use the `package:web_socket` testing library. 
https://pub.dev/documentation/web_socket/latest/testing/

If possible writing directly against `package:web_socket` may be a better option than writing against `package:web_socket_channel`.